dashboard/views/signup.ejs

```diff
@@ -75,4 +80,10 @@
     </div>
   </div>
 </div>
-<%- include('includes/footer') %>
+<script>
+  $(document).ready(function() {
+    passwordStrength("password", "password-box");
+    confirmPassword("password-box", "confirm-password");
+  }); 
+</script>
+<%- include('includes/footer') %>
```

Comment:
No newline at end of the file.

Reply:
There  are no new lines at the end of any ejs files I checked it just now. Should I add them ??

Reply:
Yes